bgpd/rfapi/vnc_debug.c

```diff
@@ -95,6 +95,7 @@ DEFUN (no_debug_bgp_vnc,
        "<no debug|undebug> bgp vnc <rfapi-query|import-bi-attach|import-del-remote|verbose>",
```

Comment:
Why are we deviating from the 'no' convention by adding 'undebug' here? Does there really need to be more than one way to turn off debugging that only works for bgp vnc?